### PR TITLE
Bump: ledger v0.9.3

### PR DESCRIPTION
**Pull request by Galadrin**
- Upgrade Ledger module to v0.9.3 to support latest Ledger model.
- Enable ledger support in binary.

Signed-off-by: dpierret <dapie@cros-nest.com>

### CODE REVIEW

**Comment by bdeneux** [comment]
Thanks for contributing @Galadrin ! ☺️. 

Adding ledger support require to add `linux-headers` package dependency on the docker container to make it working. 
https://github.com/okp4/okp4d/blob/main/Dockerfile#L12
```Dockerfile
...
# hadolint ignore=DL4006
RUN set -eux \
    && apk add --no-cache ca-certificates=20220614-r0 build-base=0.5-r3 git=2.36.2-r0 linux-headers=5.16.7-r1 \
...
```

We could also follow `cosmos-sdk` to integrate ledger support depending on the platform : https://github.com/cosmos/cosmos-sdk/blob/main/Makefile#L24-L45.

**Comment by Galadrin** [comment]
Yes, I have the docker build running to check

**Comment by ccamel** [comment]
@Galadrin thanks for the cotribution. I've activated the CI on your PR so that you can follow the workflow.

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,12 @@
 #--- Build stage
 FROM golang:1.19-alpine3.16 AS go-builder
 
+# install Linux header and build-base for native C codes
+RUN apk add --no-cache \
+    ca-certificates \
+    build-base \
+    linux-headers
```
Comment (amimart) [inline Dockerfile:8]: `ca-certificates` and `build-base` packages are already installed lower, I propose to add `linux-headers` with them and remove this `RUN` instruction.

Regarding the `linux-headers` package please reference its version (i.e. `linux-headers=5.16.7-r1`), the linter will appreciate 😉

**Review by amimart — CHANGES_REQUESTED** [review]
Thank you for bringing this, it looks almost good to me, just a little remark to make the linter happy.

**Review by amimart — APPROVED** [review]
Seems good to me thank you!

**Review by ccamel — APPROVED** [review]

**Pull request merged**